video: VP9 fallback + BYO H.264 + PyAV 15 bump (#1999)
NVIDIA/NeMo-Curator: tutorials/video/getting-started/README.md

Reviewer: ```suggestion
# Get Started with Video Curation
```

Reviewer: ```suggestion
This example extends the example above and adds an embedding stage using the `cosmos-embed1-224p` model. Use `--model-dir "$MODEL_DIR"` if the model is already downloaded.
```

  --transnetv2-min-length-s 2.0 \
  --transnetv2-max-length-s 10.0 \
  --embedding-algorithm cosmos-embed1-224p \
  --transcode-encoder h264_nvenc \
  --verbose
```

> This example demonstrates a more advanced workflow than the minimal example by using scene-aware splitting with the TransNetV2 algorithm (which detects scene boundaries instead of fixed intervals), applies the Cosmos-Embed1 embedding model to each clip, transcodes the output using the `h264_nvenc` encoder, and enables verbose logging for more detailed output. On GPUs without NVENC (such as A100/H100), pass `--transcode-encoder libvpx-vp9` instead — VP9 is a royalty-free CPU encoder that produces clips in the same `.mp4` container.

Reviewer: ```suggestion
This example demonstrates a more advanced workflow than the minimal example by using scene-aware splitting with the TransNetV2 algorithm (which detects scene boundaries instead of fixed intervals), applying the Cosmos-Embed1 embedding model to each clip, transcoding the output using the `h264_nvenc` encoder, and enabling verbose logging for more detailed output. On GPUs without NVENC (such as A100/H100), pass `--transcode-encoder libvpx-vp9` instead — VP9 is a royalty-free CPU encoder that produces clips in the same `.mp4` container.
```
